docusaurus/docs/getting-started/conventions.md

```diff
@@ -0,0 +1,62 @@
+---
+id: conventions
+title: Conventions
+sidebar_label: Conventions
+---
+
+This boilerplate establishes a convention for structuring files that favours co-location of assets and their requesters.
+
+We also use a naming convention to easily co-locate data files, which we explain further below.
+
+## Folder structure
+
+```
+├── server.js
+├── pages
+│   ├── _app.js
+│   └── index.js
+└── www
+    ├── app
+    ├── pages
+    │   └── home
+    │   │   ├── index.js
+    │   │   ├── Home.js
+    │   │   ├── Home.test.js
+    │   │   ├── Home.data.js
+    │   │   ├── Home.css
+    │   │   └── components
+    │   └── ...
+    └── shared
+        └── components
+        │   └── ...
+        └── media
+        │   └── favicons
+        │  	└── fonts
+        │   └── images
+        └── styles
+```
+
+- `pages`: This folder is necessary for Next.js since we're taking advantage of its file system routing, and this is where Next.js will search for files to route to by default.
+- `www`: Where the code for your application will be.
+    - `app`: Where your App component will be.
+    - `pages`: Where you can store the source code of your pages. This means that every file in `/pages` will import a component from here. We use this method since we only use `/pages` for routing and `www` for code that will be compiled.
+        - `home`: This an example of a component and the structure you'll find inside. Commonly, you'll find the following files here:
+            - `index.js`: This file will simply export your component file.
+            - `Home.js`: This is your component file.
+            - `Home.test.js`: This is the test file corresponding to this component.
+            - `Home.data.js`: This is where you'll find data that will be used by this component. You can find a more in depth explanation of this file convention further below.
+            - `Home.css`: This is where you'll have styles that are only used in this component.
+            - `components`: If you have abstracted parts of this component into smaller components, use this folder to co-locate them.
+    - `shared`: Where you can put content that is shared between pages and cannot be directly co-located with its interested parties.
+        - `components`: The folder for components that are shared between pages. You can find an example just above of what files each component is expected to have.
+        - `media`: The folder for media (images, fonts, etc.) that is shared between many components.
+            - `favicons`: This is where you can store your favicons.
+            - `fonts`: This is where you can store your font files. **Keep in mind**, this boilerplate only has support for `.woff2` font files!
+            - `images`: This is where you can store image files.
```

Comment:
Shouldn't we also specify what support we have in the boilerplate for images?

Reply:
I will create an issue for this: https://github.com/moxystudio/next-with-moxy/issues/33